packages/cosmic-swingset/src/launch-chain.js

```diff
@@ -51,6 +54,22 @@ import { parseLocatedJson } from './helpers/json.js';
 const console = anylogger('launch-chain');
 const blockManagerConsole = anylogger('block-manager');
 
+/**
+ * @param {{ info: string } | null | undefined} upgradePlan
+ * @param {string} prefix
+ */
+const parseCoreProposalsFromUpgradePlan = (upgradePlan, prefix = '') => {
+  const { info: upgradeInfoJson = null } = upgradePlan || {};
+
+  const upgradePlanInfo =
+    upgradeInfoJson &&
+    parseLocatedJson(upgradeInfoJson, `${prefix}upgradePlan.info`);
```

Comment:
Nit so that the caller doesn't have to compute prefix strings with proper spacing
```suggestion
    parseLocatedJson(upgradeInfoJson, `${prefix ? `${prefix} ` : ''}upgradePlan.info`);
```